fcrepo-http-api/src/test/java/org/fcrepo/integration/http/api/FedoraLdpIT.java

```diff
@@ -2067,21 +2054,6 @@ public void testIngestWithRDFLang() throws IOException {
         }
     }
 
-    @Test
-    // TODO It's not clear what this test is actually testing, or why it sleeps while running
```

Comment:
For reference, this test was added back in [August 2014](https://github.com/fcrepo4/fcrepo4/commit/db1180e899c3ad9bbb65dc2f226c86b1417df7d1) related to this [PivotalTracker ticket](https://www.pivotaltracker.com/n/projects/684825/stories/77492702).

Seems it is disabled by default and so has not been run possibly ever.